plugins/nodes/src/js/pages/nodes/nodes-grid/NodesGridContainer.js

```diff
@@ -21,9 +22,8 @@ class NodesGridContainer extends mixin(StoreMixin, QueryParamsMixin) {
     this.state = {
       filteredNodes: [],
       filters: {health: 'all', name: '', service: null},
-      hasLoadingError: false,
+      mesosXHRError: null,
       hiddenServices: [],
-      mesosStateErrorCount: 0,
```

Comment:
I thought about this for about 5 mins when I decided to change it. We should really make a global change but I'm reluctant to start that because it's scope will grow fast. For these components, IMO we are OK to just show the error when we first get an error. We still keep polling requests so if one succeeds then the component will update and remove the error screen. I think it's far more important to get this feature in than embark on a larger tech-debt overhaul for API error handling. I have created a JIRA to track the tech-debt (which will be wiped out when we have GraphQL). https://jira.mesosphere.com/browse/DCOS_OSS-740